doc/FDROID.md

```diff
@@ -2,6 +2,44 @@
 
 This document describes how to update Status APK builds for the [F-Droid](https://f-droid.org/) Android application catalogue.
 
+
+# Automated release
+
+## Pre-requisites
+
+- A gitlab account
+- A forked copy of the [frdroiddata repo](https://gitlab.com/fdroid/fdroiddata) on your gitlab account.
+- A link to the release published
+- `apkanalyzer` installed
+
+## Steps
+
+
+### Checkout  the release/tag/hash you want to release
+
+First checkout the branch/commit you want to release
+
+For example:
+
+`git checkout release/1.13.x` 
+
+To checkout a release branch.
+
+Or if you want to release a specific commit:
+
+`git checkout ${hash}`
+
+### Run the script
+
+Run the script making sure you pass your USERNAME and RELEASE_LINK in the env variables:
+
+For example:
+
+`env USERNAME=cammellos RELEASE_LINK="https://status-im-releases.ams3.digitaloceanspaces.com/StatusIm-210426-081458-896b58-release-universal.apk" bash scripts/release-fdroid.sh`
```

Comment:
For one, not sure why you use `env`, since you can just pass variable as-is in bash:
```
 > SOME_VAR="wtf" bash -c 'echo "SOME_VAR: $SOME_VAR"' 
SOME_VAR: wtf
```
And if your script has a proper shebang like `#!/usr/bin/env bash` the you don't need the `bash` part either.

And if you make this into a `Makefile` target you can simplify it like this:
```Makefile
fdroid-pr: TARGET := android
fdroid-pr: GL_USERNAME ?= "${USER}"
fdroid-pr: ##@fdroid Create F-Droid release PR
    scripts/release-fdroid.sh
```
And then just:
```sh
 > make fdroid-pr RELEASE_LINK="https://..."
```

Reply:
>  > SOME_VAR="wtf" bash -c 'echo "SOME_VAR: $SOME_VAR"' 
SOME_VAR: wtf
I use the fish shell, so I need to use `env` ;) 
I'll create a make file target

Reply:
But if you run it via makefile using nix shell it won't be using fish but bash.